doc/src/explanation/gotchas.rst

```diff
@@ -7,7 +7,7 @@ Gotchas and Pitfalls
 .. role:: input(strong)
 
 Introduction
-============
+--------------
```

Comment:
This doesn't actually do anything, because of the way rst works. We should revert the header changes in this file.